File: launchdarkly/csharp/src/OSEG.LaunchDarklyExamples/DeleteModelConfigExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Org.LaunchDarklyTools.Api;
using Org.LaunchDarklyTools.Client;
using Org.LaunchDarklyTools.Model;

namespace OSEG.LaunchDarklyExamples;

public class DeleteModelConfigExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey.Add("Authorization", "YOUR_API_KEY");

        try
        {
            new AIConfigsBetaApi(config).DeleteModelConfig(
                lDAPIVersion: null,
                projectKey: "default",
                modelConfigKey: null
            );
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling AIConfigsBetaApi#DeleteModelConfig: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: launchdarkly/typescript-axios/src/DeleteModelConfigExample.ts
import axios, { AxiosError } from "axios";
import * as api from "launchdarkly_client"

const configuration = new api.Configuration({
  apiKey: "YOUR_API_KEY",
});

new api.AIConfigsBetaApi(configuration).deleteModelConfig(
  "beta", // lDAPIVersion
  "default", // projectKey
  "modelConfigKey_string", // modelConfigKey
).catch((error: Error | AxiosError) => {
  console.log("Exception when calling AIConfigsBetaApi#deleteModelConfig:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
